packages/api/src/filestorage/file/services/googledrive/index.ts

```
@@ -74,108 +175,336 @@
+    permissionIds: string[],
+    files: GoogleDriveFileOutput[],
+    drive: ReturnType<typeof google.drive>,
+  ): Promise<GoogledrivePermissionOutput[]> {
+    const permissionIdToFiles = new Map<string, string[]>();
+
+    for (const file of files) {
+      if (file.permissionIds?.length) {
+        for (const permissionId of file.permissionIds) {
+          if (permissionIdToFiles.has(permissionId)) {
+            // only need one file_id to get the permission
+            continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)